components/cnc-tech/3020-xx-0300-00.stanza

```diff
@@ -0,0 +1,103 @@
+#use-added-syntax(jitx)
+defpackage ocdb/cnc-tech/_3020-xx-0300-00:
+  import core
+  import collections
+  import jitx
+  import jitx/commands
+  import ocdb/defaults
+  import ocdb/design-vars
+  import ocdb/land-patterns
+  import ocdb/tolerance
+  import ocdb/box-symbol
+  import ocdb/rules
+  import ocdb/generic-components
+  import ocdb/symbols
+  import ocdb/land-protrusions
+
+pcb-landpattern lp-3020-xx-0300-00 (number-of-leads:Int, a-dim:Double):
+  make-n-pin-soic-landpattern(number-of-leads, 2.54, tol(9.5, 0.15), tol(8.75, 0.2), tol(a-dim, 0.35), tol((9.5 - 2.54) / 2.0, 0.15), tol(0.64, 0.01), "row-major" )
+;   defn make-n-pin-soic-landpattern  (num-pins:Int,
+;                                              pitch:Double,               ; e
+;                                                   lead-span:Toleranced,       ; E
+;                                                                    package-length:Toleranced,  ; E1
+;                                                                                    package-width:Toleranced,   ; D
+;                                                                                                      terminal-length:Toleranced, ; L
+;                                                                                                                                      terminal-width:Toleranced)  ; b
+
+
+public pcb-component component (number-of-leads:Int): 
+; datasheet https://www.cnctech.us/pdfs/3020-XX-0300-00%20(9T).pdf
+  var desc = "CONN"
+  var a-dim = 1.0
+  var b-dim = 1.0
+  var c-dim = 1.0
+  var pn = "3020-XX-0300-00"
+  switch(number-of-leads) :
+    6:
+      a-dim = 15.24
+      b-dim = 5.08
+      c-dim = 12.94
+      pn = "3020-06-0300-00"
+      desc = "CONN HEADER SMD 6POS 0.1 IN 2.54MM SHROUDED"
+    8:
+      a-dim = 17.78
+      b-dim = 7.62
+      c-dim = 15.48
+      pn = "3020-08-0300-00"
+      desc = "CONN HEADER SMD 8POS 0.1 IN 2.54MM SHROUDED"
```

Comment:
likewise:
`data = get-data-for-8-leads()`

Reply:
fixed